### PR TITLE
Starting out on a service function to listen for new Fennel Signals.

### PR DESCRIPTION
**Pull request by Romulus10**

### CODE REVIEW

**Comment by Romulus10** [comment]
@isaacadams I know for a fact that this is throwing an illegal constructor error in the Feed page. It's also not finished, I wanted to make sure it was querying the node correctly before I started building up the feed display logic. However, if you have a second could you take a glance at that page at let me know if there's something stupid I'm missing that's causing the illegal constructor error?

File: src/services/Node/index.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import {ApiPromise, WsProvider} from '@polkadot/api';
 import axios from 'axios';
-import { NODE_URI_HTTP } from '../../config';
+import {BehaviorSubject} from 'rxjs';
+import {NODE_URI_HTTP} from '../../config';
 
 class Node {
+  _signals = new BehaviorSubject([]);
+
+  $signals$ = this._signals.asObservable();
```
Comment (isaacadams) [inline src/services/Node/index.js:9]:
```suggestion
  signals$ = this._signals.asObservable();
```

**Comment by isaacadams** [comment]
> @isaacadams I know for a fact that this is throwing an illegal constructor error in the Feed page. It's also not finished, I wanted to make sure it was querying the node correctly before I started building up the feed display logic. However, if you have a second could you take a glance at that page at let me know if there's something stupid I'm missing that's causing the illegal constructor error?

was missing an import -- easy fix, so I just committed it

**Comment by Romulus10** [comment]
Of course that's all it was. 😂 Thanks @isaacadams.

**Comment by Romulus10** [comment]
Everything in here still needs some work, but the Substrate work is at least workable, so I wanted it up there sooner rather than later.

File: src/views/pages/NewFeedMessage/Index.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 import PageContainer from '../../components/PageContainer';
 import PageTitle from '../../components/PageTitle';
-import Text from '../../components/Text';
 import Button from '../../components/Button';
 import FeedSubNav from '../../components/FeedSubNav';
+import Node from '../../../services/Node';
+import {useServiceContext} from '../../../contexts/ServiceContext';
 
 function NewFeedMessage() {
+  const {_, keymanager} = useServiceContext();
```
Comment (isaacadams) [inline src/views/pages/NewFeedMessage/Index.js:10]: I think `const {keymanager} = ..` will suffice without the `_`

Comment (Romulus10) [reply src/views/pages/NewFeedMessage/Index.js:10]: Won't that get back both the messageService _and_ the KeyManager?

**Pull request merged**